modules/rss/src/test/java/org/mule/module/rss/transformers/RssParserTestCase.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.rss.transformers;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.mule.tck.junit4.AbstractMuleTestCase;
+import org.mule.tck.size.SmallTest;
+
+import com.rometools.rome.feed.synd.SyndCategory;
+import com.rometools.rome.feed.synd.SyndEntry;
+import com.rometools.rome.feed.synd.SyndFeed;
+
+import java.io.InputStream;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+@SmallTest
+public class RssParserTestCase extends AbstractMuleTestCase
+{
+
+    private List<SyndEntry> entries = null;
+
+    @Before
+    public void setup() throws Exception
+    {
+        ObjectToRssFeed objectToRssFeed = new ObjectToRssFeed();
+        InputStream fileInputStream;
+        fileInputStream = getClass().getResourceAsStream("/rss-test.xml");
+        SyndFeed feed = (SyndFeed) objectToRssFeed.doTransform(fileInputStream, null);
+        entries = feed.getEntries();
+    }
+
+    @Test
+    public void testParsedEntries() throws Exception
+    {
+        assertValues(entries.get(0));
```

Comment:
Add some error message to use when the assertion fail or extract each line to a method so we can understand what are you asserting without having to read the RRS file